packages/web-components/src/components/inline-loading/__tests__/inline-loading-test.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright IBM Corp.  2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { expect, fixture, html, oneEvent } from '@open-wc/testing';
+import '@carbon/web-components/es/components/inline-loading/index.js';
+
+describe('cds-inline-loading', function () {
+  const inlineLoading = html`<cds-inline-loading></cds-inline-loading>`;
+
+  it('should render', async () => {
+    const el = await fixture(inlineLoading);
```

Comment:
This test just creates an element but doesn't verify anything
```suggestion
    const el = await fixture(inlineLoading);
    expect(el).to.exist; 
```